pkg/gadgets/top/file/types/types.go

```diff
@@ -90,7 +90,7 @@ type Stats struct {
 	MountNsID  uint64 `json:"mountnsid,omitempty"`
 	Filename   string `json:"filename,omitempty"`
 	Comm       string `json:"comm,omitempty"`
-	FileType   byte   `json:"fileType,omitempty"`
+	FileType   byte   `json:"fileType,omitempty"` // R = Regular File, S = Socket, O = Other
```

Comment:
>Is it right when it is in the same line for such a small comment?
>
IMO, it is ok for this PR. However, this struct should be self-descriptive, and we shouldn't need this comment. To solve that, I agree we need to use an `enum` or something similar to restrict the values `FileType` can take, and it should be synch with the BPF program. Of course, you can manage it in another PR 🙂 .

Reply:
Is there a way to do that?
Like defined an enum in a C header and "import" these values?